src/Compilers/CSharp/Test/Emit2/Semantics/CollectionExpressionTests.cs

```diff
@@ -35119,10 +35237,8 @@ .locals init (System.Collections.Generic.List<int> V_0,
                 """);
         }
 
-        [Theory]
-        [InlineData(TargetFramework.Net80)]
-        [InlineData(TargetFramework.Standard)]
-        public void List_SingleSpread_CustomCollection_ICollectionAndStructEnumerator(TargetFramework targetFramework)
```

Comment:
[The original motivation](https://github.com/dotnet/roslyn/pull/74630#discussion_r1717240355) for verifying multiple target frameworks was to test `AddRange` code path. Since now there is no difference between codegen on .NET Framework and modern .NET, I don't see value in having multiple TFMs to verify against